src/main/java/Duke.java

```diff
@@ -1,10 +1,206 @@
+import java.io.*;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    private static final String FILE_PATH = "./data/taskCategory.txt";
+
+    static ArrayList<Task> listOfItems = new ArrayList<>();
+
+    //variable stores the number of tasks being added
+    static int sizeOfAddedItems = 0;
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+
```

Comment:
The main method is doing a lot of things. please follow the instructions in A-MoreOOP and refactor the code accordingly.